src/app/services/postman-sms/postman-sms.service.ts

```diff
@@ -88,8 +96,67 @@ class PostmanSmsService {
    *
    * SMSes will be sent using FormSG sender id.
    */
-  private sendInternalSms() {
-    // stub
+  _sendInternalSms(
```

Comment:
sendInternalSms seems largely similar to `sendMopSms`. Why not implement a generic sendSms function that takes in different campaignID and apikey instead? (i.e. dependency inversion)

Reply:
It was designed to be very explicit that this function (or what the developer is about to do) is to carry the intention of sending to internal users.

Having them separated as functions aims to improve the readability during implementation, as it is a core product decision to make, over DRY principles.